### DISCUSSION TITLE
Maven Repo lists unreleased Quarkus versions

### DISCUSSION THREAD

**Discussion by JuJup** [quarkusio/quarkus#44617] [Q&A]
Hello!

I stumbled upon that before and I still don't know why that happens:
In the Maven Repo sometimes I see older versions of the parent plugin, than the currently released latest version.

To be concrete:
Right now, [3.16.4](https://github.com/quarkusio/quarkus/releases/tag/3.16.4) seems to be the latest version. There is also a release candidate: [3.17.0.CR1](https://github.com/quarkusio/quarkus/releases/tag/3.17.0.CR1).

However, in Maven, you can [find also 3.17.0](https://repo.maven.apache.org/maven2/io/quarkus/io.quarkus.gradle.plugin.parent/3.17.0/) as a "normal" version. Why is this happening?

Ok, so what's the problem about that:
When using Dependency Update Bots like Renovate, it checks Maven for new Releases and finds e.g. for `io.quarkus-platform:quarkus.gradle.plugin` version 3.16.4 and for `io.quarkus.platform:quarkus-bom` version 3.17.0.
This leads to errors at different steps, I guess they better should be equal.

**Comment by gsmet** [comment]
Hi!

A Quarkus release is composed of two things:

- The Quarkus core release: it's the content of this repository.
- The Quarkus Platform release: it aligns some additional components such as Camel Quarkus, Quarkus CXF and some others into what we call the Platform BOM i.e. `io.quarkus.platform:quarkus-bom`

Most of the time, these releases are made in a row but for the first release of each stream, we release the core first and the Platform one week after so that the Platform members can push new releases to fix potential compatibility issues.

We released the core 3.17.0 this Wednesday and the corresponding Platform release will be made next Wednesday.

So for any new stream, you have a one week delay between the `io.quarkus` artifacts availability and the `io.quarkus.platform` artifacts availability which is what you experience here.

We solved this issue for the Maven plugin as we repackage it with the Platform but we couldn't find a good solution for the Gradle one.

Now, in your case, wouldn't using a common property for both versions fix your issue with Renovate?

**Reply by JuJup** [reply]
Thanks for the clarification!
Yes, using a common property will probably solve the problem on the client side. I will try and let you know.

**Reply by JuJup** [reply]
No, that's unfortunately not working (at least with Renovate). For anyone with the same problem, [there's an Issue in Renovate](https://github.com/renovatebot/renovate/issues/9739) with the idea of syncing versions in groups.
